test/e2e/test/logstash/http_client.go

```diff
@@ -31,9 +32,16 @@ func NewLogstashClient(logstash v1alpha1.Logstash, k *test.K8sClient) (*http.Cli
 }
 
 func DoRequest(client *http.Client, logstash v1alpha1.Logstash, method, path string) ([]byte, error) {
-	scheme := "http"
+	var scheme = "http"
+	var port = network.HTTPPort
+	for _, service := range logstash.Spec.Services {
+		if service.Name == "api" {
```

Comment:
nit: maybe `"api"` would deserve to be a constant that could be reused here, in `pkg/controller/logstash/service.go` and `pkg/controller/logstash/pod.go`?